app/frontend/src/components/objectList/ObjectPermissions.vue

```diff
@@ -9,8 +9,7 @@
     >
       <template #title>
         <v-icon color="primary" class="mr-5"> mdi-account-group </v-icon>
-        Permissions for <br />
-        {{ objectName }}
+        Permissions for<span class="ml-1 font-weight-bold"> {{ objectName }}</span>
```

Comment:
Stylistic thing, but it makes more sense to me to have the space after 'for' instead of within the <span>?

Reply:
I agree with Kyle on this. Most other places in the code keep the space outside the child in-line element, so we should stay consistent with that.